### ISSUE TITLE
Adoption

### ISSUE THREAD

**Issue by Infiziert90** [PunishedPineapple/ReadyCheckHelper#8]
Just wanted to ask if I adopt this plugin and implement the open issues?

**Comment by PunishedPineapple** [comment]
If you want to try to navigate the trash I created, it's up for adoption.  I don't really care who takes it as long as they know what they're getting themselves into first.  A few notes for any potential adopter, though:

- The branch that was getting released is [Legacy_GUI](https://github.com/PunishedPineapple/ReadyCheckHelper/commits/Legacy_GUI).  This is because I had started to do work to use icon nodes in the party list in the main branch, but I never got it working quite right:
- The icon nodes in the party list would affect or be affected by other places where the ready check texture was used (like in the `/isearch` window, or when selling cards at the Gold Saucer).  I could never figure out why this was happening.  Other than that, it was functional in the party list, but nodes had not yet been added to the alliance or cross-world alliance lists before I gave up.
- Should you choose to revert what I did with nodes on the main branch and start over yourself, there is one important part of a commit that should be preserved; in the functions to traverse the nodes to find where to draw the icons, I had previously used `NodeListSize` instead of `NodeListCount`, which I believe was the cause of a crash related to the cross-world alliance list.  See [this commit](https://github.com/PunishedPineapple/ReadyCheckHelper/commit/ac163f11c7fc31c66283cbc79d5a9a979f03e4e5#diff-d51af3af70172129027ed7e10cb6e9b049fc55f378c9559bb3f2558a40b02835R610).
- At some point after I first made the plugin, someone added the ready check agent to ClientStructs.  If you make major changes, it may or may not be worth replacing my memory reading with that at the same time.
